internal/pkg/template/templates/environment/cf.yml

```diff
@@ -174,7 +174,7 @@ Resources:
       Protocol: HTTP
   HTTPSListener:
     Type: AWS::ElasticLoadBalancingV2::Listener
-    DependsOn: HTTPSCert
+    DependsOn: HTTPSCertArn
```

Comment:
Why does the listener depend on the output `HTTPSCertArn` 🤔

Reply:
it was a typo. Sorry.